awx_collection/plugins/modules/ad_hoc_command.py

```diff
@@ -158,6 +159,8 @@ def main():
     # Attempt to look up the related items the user specified (these will fail the module if not found)
     post_data['inventory'] = module.resolve_name_to_id('inventories', inventory)
     post_data['credential'] = module.resolve_name_to_id('credentials', credential)
+    if execution_environment:
+        post_data['execution_environment'] = module.resolve_name_to_id('execution_environments', execution_environment)
```

Comment:
Can we add an test in `awx_collection/tests/integration/targets/ad_hoc_command/tasks/main.yml` for this?

Reply:
@john-westcott-iv Thank you for reviewing. I added integration test.